index.html

```diff
@@ -80,20 +81,20 @@ <h2>services</h2>
         freegan, viral lumbersexual ennui kinfolk. Twee fixie bitters pop-up, roof party deep v 3 wolf moon irony etsy
         man braid fap jean shorts.
       </p>
-      <img src="assets/img.jpg" alt="Hello Studio Office">
+      <img src="assets/img.jpg" alt="Hello Studio Office" class="services__image">
     </section>
-    <section>
+    <section class="contact">
       <h2>contact</h2>
-      <form>
-        <label for="name">name</label>
-        <input type="text" id="name">
-        <label for="topic">topic</label>
-        <textarea id="topic"></textarea>
-        <label for="email">email</label>
-        <input type="email" id="email">
-        <button>send</button>
+      <form class="contact-form">
+        <label for="name" class="contact-form__label">name</label>
```

Comment:
Te pary label + input warto wrzucić w jakiegoś diva, często to jest potrzebne żeby to dobrze ostylować. Wtedy się super układają nazwy, bo można mieć `form-field`, a w nim `form-field__label` i `form-field__input`.